plugins/slo/pkg/slo/openslo.go

```diff
@@ -99,17 +101,14 @@ func ParseToIndicator(slo *api.ServiceLevelObjective, service *api.Service, ctx
 	return &SLI, err
 }
 
-func ParseToObjectives(slo *api.ServiceLevelObjective, ctx context.Context, lg hclog.Logger) []oslov1.Objective {
-	objectives := make([]oslov1.Objective, 0)
-	for i, target := range slo.GetTargets() {
-		newObjective := oslov1.Objective{
-			DisplayName:     slo.GetName() + "-target" + strconv.Itoa(i),
-			Target:          float64(target.GetValueX100()) / 100,
-			TimeSliceWindow: slo.GetBudgetingInterval(),
-		}
-		objectives = append(objectives, newObjective)
+func ParseToObjective(slo *api.ServiceLevelObjective, ctx context.Context, lg hclog.Logger) oslov1.Objective {
+	target := slo.GetTarget()
+	newObjective := oslov1.Objective{
+		DisplayName:     slo.GetName() + "-target",
+		Target:          float64(target.GetValueX100()) / 100,
```

Comment:
I'd recommend adding this math into a helper function added to the Target type (inside the apis/slo package). like GetValueNormalized or something

Reply:
Done